_layouts/page-breadcrumbs.html

```diff
@@ -9,6 +9,18 @@
   
     {% when '1-topic-landing' or 'alt-1-topic-landing' %}
       {% include 1-topic-landing.html %}
+
+    {% when '2-detailed-landing' %}
+      {% include 2-detailed-landing.html %}
+
+    {% when '4-action-page-pending' %}
```

Comment:
Lines 16 - 23 are temporary placeholders. They'll be replaced with includes as I build out these templates. It's an effort to keep merge conflicts to a minimum if I end up stacking a series of pull requests.